packages/lesswrong/lib/collections/users/formGroups.ts

```diff
@@ -26,58 +28,58 @@ export const formGroups: Partial<Record<string,FormGroupType>> = {
   privacy: {
     order: 16,
     name: "privacy",
-    label: "Privacy Settings",
+    label: preferredHeadingCase("Privacy Settings"),
     startCollapsed: true,
   },
   adminOptions: {
     name: "adminOptions",
     order: 25,
-    label: "Admin Options",
+    label: preferredHeadingCase("Admin Options"),
     startCollapsed: true,
   },
   paymentInfo: {
     name: "paymentInfo",
-    label: "Prize/Payment Info",
+    label: preferredHeadingCase("Prize/Payment Info"),
     order: 35,
     startCollapsed: false,
   },
   disabledPrivileges: {
     order:40,
     name: "disabledPrivileges",
-    label: "Disabled Privileges",
+    label: preferredHeadingCase("Disabled Privileges"),
     startCollapsed: true,
   },
   banUser: {
     order:50,
     name: "banUser",
-    label: "Ban & Purge User",
+    label: preferredHeadingCase("Ban & Purge User"),
     startCollapsed: true,
   },
   moderationGroup: {
     order:60,
     name: "moderation",
-    label: "Moderation & Moderation Guidelines",
+    label: preferredHeadingCase("Moderation & Moderation Guidelines"),
     startCollapsed: true,
   },
   aboutMe: {
     name: 'aboutMe',
     order: 100,
-    label: 'About Me'
+    label: preferredHeadingCase('About Me'),
```

Comment:
```suggestion
    label: isEAForum ? "About you" : "About Me",
```
Realised we refer to you as "me" in this specific instance (I prefer "you" in products, like "Your activity")